File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int,int> contains;
        for(auto &elem : nums){
            if(contains[elem] > 0)
                return true;
            ++contains[elem];
        }
        return false;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set(nums)
        longest, length = 0,1
        for elem in nums:
            if elem - 1 not in s:
                length = 1
                while length + elem in s:
                    length += 1
                longest = max(longest, length)
        return longest
                
